### PR TITLE
Add R package microseq

### PR DESCRIPTION
**Pull request by mdehollander**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/r-microseq/build.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# R refuses to build packages that mark themselves as Priority: Recommended
+mv DESCRIPTION DESCRIPTION.old
+grep -v '^Priority: ' DESCRIPTION.old > DESCRIPTION
+
+$R CMD INSTALL --build .
+
+# Add more build steps here, if they are necessary.
+
```
Comment (bgruening) [inline recipes/r-microseq/build.sh]: Can you remove the comments please-

File: recipes/r-microseq/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-microseq
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: microseq_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/microseq_{{ version }}.tar.gz
```
Comment (bgruening) [inline recipes/r-microseq/meta.yaml]: Can you please checkout latest master and use the latest bioconductor skeleton. We will get a third URL with the new script.

File: recipes/r-microseq/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+requirements:
+  build:
+    - r-base
+    - r-rcpp >=0.11.1
+    - gcc
+
+  run:
+    - r-base
+    - r-rcpp >=0.11.1
+    - libgcc
```
Comment (bgruening) [inline recipes/r-microseq/meta.yaml]: This is only needed for linux `# [linux]`

File: recipes/r-microseq/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+  summary: Basic functions for microbial sequence data analysis.
+  license_family: GPL2
+
+# The original CRAN metadata for this package was:
+
+# Package: microseq
+# Type: Package
+# Title: Basic Biological Sequence Analysis
+# Version: 1.2
+# Date: 2017-01-13
```
Comment (bgruening) [inline recipes/r-microseq/meta.yaml]: remove comments.

File: recipes/r-microseq/meta.yaml (hunk 2 above)
Comment (mdehollander) [inline recipes/r-microseq/meta.yaml]: This package is not in bioconductor, but in CRAN. I created the `meta.yaml` it quickly using `conda skeleton cran microseq`. What should be different?

Comment (bgruening) [inline recipes/r-microseq/meta.yaml]: Gnarf, sorry, to much R in the last days. Everything is fine.

**Comment by mdehollander** [comment]
@bgruening I need some help building the package for OSX, because the build fails: https://travis-ci.org/mdehollander/bioconda-recipes/jobs/239958094. With this message:
```

ERROR:conda.core.link:An error occurred while installing package 'r::r-base-3.3.1-0'.

PaddingError: Placeholder of length '80' too short in package /anaconda/conda-bld/r-microseq_1496755182282/_b_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/bin/Rscript.

The package must be rebuilt with conda-build > 2.0.

Attempting to roll back.
```

I have seen this error in https://github.com/bioconda/bioconda-recipes/issues/4321 and https://github.com/bioconda/bioconda-recipes/pull/4531, but I am not sure how to fix it

**Comment by mdehollander** [comment]
Strange, building it again works now for both osx and linux: https://travis-ci.org/mdehollander/bioconda-recipes

**Comment by bgruening** [comment]
ok, strange but good. Do you want me to merge this PR into our bulk branch an rebuild it against 3.2.2? Or do you need 3.2.1?

File: recipes/r-microseq/build.sh (hunk 1 above)
Comment (mdehollander) [inline recipes/r-microseq/build.sh]: done

File: recipes/r-microseq/meta.yaml (hunk 3 above)
Comment (mdehollander) [inline recipes/r-microseq/meta.yaml]: added it

File: recipes/r-microseq/meta.yaml (hunk 4 above)
Comment (mdehollander) [inline recipes/r-microseq/meta.yaml]: done

**Comment by mdehollander** [comment]
Now I check it I see it needs at least 3.3.1 (https://cran.r-project.org/web/packages/microseq/). I am not an expert on this package, but I guess it should also work with 3.2.2. You can decide what is best.

**Comment by bgruening** [comment]
added here: https://github.com/bioconda/bioconda-recipes/commits/bulk

**Comment by bgruening** [comment]
This will now take a while, but hopefully tm morning you have your package at least for Linux.

**Pull request closed without merging**